koku/cost_models/view.py

```diff
@@ -142,10 +145,12 @@ def get_queryset(self):  # noqa: C901
                     LOG.error(queryset_error)
         return queryset
 
+    @never_cache
     def create(self, request, *args, **kwargs):
         """Create a rate."""
         return super().create(request=request, args=args, kwargs=kwargs)
 
+    @vary_on_headers(RH_IDENTITY_HEADER)
```

Comment:
Can we invalidate the list/retreieve cache if someone does a create, update, or delete operation?

Reply:
Doug asked the same question. Short answer: Not easily enough to be worthwhile.

I had a side-chat with Doug about it and our thinking is just to `@never_cache` views where we need to show state changes like this.

Reply:
I had the same question for `/providers`.  I suggest we do the same thing here and not cache `/costmodels`